static/sb-styles/preview.css

```diff
@@ -125,4 +126,14 @@ html {
     width: 16rem;
     padding-top: 6.4rem;
     padding-bottom: 3.2rem;
+}
+
+/**
+ * View Transitions
+ */
+::view-transition-group(*),
+::view-transition-old(*),
+::view-transition-new(*) {
+    animation-timing-function: linear(0, 0.03 1.3%, 0.113 2.7%, 0.84 11.5%, 0.981 14.7%, 1.049 18.2%, 1.061 20%, 1.061 22.2%, 1.011 32.3%, 0.997 39.3%, 1);
+    animation-duration: 1s;
```

Comment:
I'm curious about these styles! Is this something we'll need to set up in `webapp` as well? And for storybook in other repos?

Reply:
This is an example for us to showcase how View Transitions works, but it would be up for consumers to define their own view transitions.

Reply:
Maybe we could explore further how to disable this in WB with the idea that we have been considering around a global context/object to determine when to animate.

Reply:
Maybe that could be part of `wonder-blocks-styles` to centralize some animations for consistency?